tests/TestCase.php

```diff
@@ -43,7 +43,7 @@ abstract class TestCase extends BaseTestCase
     protected function setUp(): void
     {
         parent::setUp();
-        $this->host = getenv('HOST');
+        $this->host = getenv('MEILISEARCH_HOST');
```

Comment:
Why change `HOST` into `MEILISEARCH_HOST` and not `API_KEY` into `MEILISEARCH_API_KEY`?

Reply:
Good catch, I missed this one!